sdk-cpp/include/do_config.h

```diff
@@ -27,4 +27,4 @@ void deliveryoptimization_free_version_buf(char** ppBuffer);
 }
 #endif
 
-#endif // (DO_PLATFORM_ID == DO_PLATFORM_ID_LINUX)
```

Comment:
>(DO_PLATFORM_ID == DO_PLATFORM_ID_LINUX) [](start = 10, length = 40)

Revert?

Reply:
I think I prefer using // Linux or // Windows rather than the full string, meaning is conveyed and its less verbose

---
In reply to: [606373122](https://github.com/microsoft/do-client/pull/19#discussion_r606373122) [](ancestors = 606373122)

Reply:
Ok but cpplint might complain (we'll deal with it later).

---
In reply to: [606388150](https://github.com/microsoft/do-client/pull/19#discussion_r606388150) [](ancestors = 606388150,606373122)